packages/scrubber-bar/demo/index.html

```diff
@@ -17,10 +25,19 @@
 
     render(
       html`
-        <scrubber-bar></scrubber-bar>
+        <scrubber-bar
+          sectionMarkerPercentages='[0, 10, 11, 25, 30, 50, 75, 100]'
+          value='37'
+          @valuechange=${valueChange}>
+        </scrubber-bar>
```

Comment:
QUESTION: So you're okay with the newline between the opening `<scrubber-bar>` and closing `</scrubber-bar>`. Same question applies down below when the closing tag is put on a different line from the opening tag.

Reply:
I think this formatting helps to distinguish the parts or the tag, but I'm open to other styles if you have ideas!